features/networking/multus-ipv6.feature

```diff
@@ -50,3 +50,64 @@ Feature: Multus-CNI ipv6 related scenarios
       | curl | -g | -6 | --connect-timeout | 5 | [<%= cb.pod1_multus_ipv6 %>]:8080 |
     Then the step should succeed
     And the output should contain "Hello OpenShift"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-38521
+  @admin
+  Scenario: IPv6 testing for OCP-31999: Whereabouts should exclude IPv6 ranges	
+  # Bug https://bugzilla.redhat.com/show_bug.cgi?id=1913062
+  # Bug https://bugzilla.redhat.com/show_bug.cgi?id=1917984
+  # Make sure that the multus is enabled
+    Given the master version >= "4.6"
+    And the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard
+    And evaluation of `node.name` is stored in the :target_node clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    Given I obtain test data file "networking/multus-cni/NetworkAttachmentDefinitions/IPv6/whereabouts-excludeIPv6.yaml"
+    When I run oc create as admin over "whereabouts-excludeIPv6.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %>  |
+    Then the step should succeed
+
+    # Create the three pods which consumes the custom resource
+    Given I obtain test data file "networking/multus-cni/Pods/IPv6/ipv6-pod.yaml"
+    When I run oc create over "ipv6-pod.yaml" replacing paths:
+      | ["spec"]["nodeName"] | "<%= cb.target_node %>" |
+    Then the step should succeed
+    When I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :describe client command with:
+      | resource | pod |
+    Then the output should contain:
+      | fd00:dead:beef:1::1 |
```

Comment:
@weliang1 hahah I used to write the pattern `0xdeadbeef` at one of my first firmware job's memory diagnostic test. :)

Reply:
to make it more English like he should've used `f00d:dead:beef:cafe::1`

Reply:
`fd00:dead:beef::` is a valid IPv6 unique local addresses and suggested by @rbbratta in https://github.com/openshift/verification-tests/pull/905.